package.json

```diff
@@ -40,18 +40,14 @@
   "dependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-etherscan": "^2.1.1",
-    "@nomiclabs/hardhat-truffle5": "^2.0.0",
-    "@nomiclabs/hardhat-web3": "^2.0.0",
-    "@openzeppelin/contracts": "3.4.0-solc-0.7",
     "@nomiclabs/hardhat-waffle": "2.0.1",
-    "@truffle/hdwallet-provider": "^1.4.1",
+    "@openzeppelin/contracts": "3.4.0-solc-0.7",
     "axios": "^0.21.1",
     "chai": "^4.3.4",
     "dotenv": "^8.2.0",
     "elliptic": "^6.5.4",
     "eslint": "^7.18.0",
     "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-truffle": "^0.3.1",
     "ethereum-waffle": "^3.3.0",
     "ethereumjs-util": "^7.0.7",
     "ethers": "^5.0.32",
```

Comment:
It looks like 5.4 is the latest version. Is 5.0.32 the version required by hardhat plug-ins, or can we use a later version?

Reply:
Yes, we could